### DISCUSSION TITLE
How to make Popup window?

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [zauberzeug/nicegui#1347] [Q&A] [answered]
### Question

Hello,
for my new project I am making an online poll, and of course I'm using NiceGUI because I really like it's approach of GUI of web, for pythonists.
I have read the documentation, but could not see anything else than `ui.notify()` for pop up.
I want users to click `OK`, then pop up will fade away. what is best solution for that?
Thanks for hints.
Best

**Comment by Shahin-rmz** [comment]
`ui.notify` has also `close_button='OK'` but it will also fade away without clicking on it after some seconds.
I do not want it to go away before clicking and the rest of the code should not work unless user clicks on `OK`.

**Comment by falkoschindler** [comment, marked as answer]
It looks like a use case for [`ui.dialog`](https://nicegui.io/documentation/dialog). Using `.props('persistent')` you can prevent the user from closing the dialog with the escape key or clicking in the background.

Alternatively you can prevent the notification from closing setting the timeout to 0:
```py
ui.notify('Hello!', timeout=0, close_button='Ok')
```

**Comment by Shahin-rmz** [comment]
Thanks so much.
exactly what I wanted.
